src/structures/Presence.js

```diff
@@ -31,10 +38,10 @@ class Presence {
 
     /**
      * The devices this presence is on
-     * @type {?object}
-     * @property {PresenceStatus} web
-     * @property {PresenceStatus} mobile
-     * @property {PresenceStatus} desktop
+     * @type {?Object}
+     * @property {PresenceStatus} web The current presence in the web application
```

Comment:
You need to use the `ClientPresenceStatus` typedef here (and the two lines below).